### DISCUSSION TITLE
Can't login, "an error occured while sending the request" (Osu!Lazer)

### DISCUSSION THREAD

**Discussion by MiRw3b** [ppy/osu#21263] [Q&A] [answered]
### Type

Other

### Bug description

So, i'm trying to login on osu!lazer and it just doesn't works, and gives me an error "an error occured while sending the request"
It used to work before, but i don't know why it just doesn't wants anymore, **while normal osu does work**.
I'm on windows 10 20H2 (atlas os) and i don't understand why i keep getting this issue, i already tried reinstalling osu!lazer and deleting almost all of its file in `Roaming\osu\` except for the skins and beatmaps, but it just doesn't

### Screenshots or videos

![osubroke](https://user-images.githubusercontent.com/60517683/201967211-8e1a0b6a-30ea-435c-948f-5c5ab1c13a41.JPG)

### Version

2022.1101.0.0

### Logs

[network.log](https://github.com/ppy/osu/files/10014271/network.log)
[performance.log](https://github.com/ppy/osu/files/10014273/performance.log)
[updater.log](https://github.com/ppy/osu/files/10014274/updater.log)
[runtime.log](https://github.com/ppy/osu/files/10014275/runtime.log)

**Comment by nagi-desuuu** [comment]
Did you check your firewall settings to make sure that nothing is blocking Lazer from sending requests?

**Comment by MiRw3b** [comment]
yeah, i also disabled the firewall directly

**Comment by peppy** [comment, marked as answer]
From the attached log, it looks like your PC is not handling SSL connections correctly. What OS version and what network environment are you running in? Do you have any proxies or otherwise sitting between you and the internet?

If none of this applies to you, your PC may be compromised. A reinstall could work well if that's something you'd consider.

**Reply by MiRw3b** [reply]
My OS version is Windows 10 (20H2), but it's a debloated version (atlasOS)
I think i disabled something about old SSL somewhere, i'm gonna check that

**Reply by MiRw3b** [reply]
> My OS version is Windows 10 (20H2), but it's a debloated version (atlasOS) I think i disabled something about old SSL somewhere, i'm gonna check that

i enabled SSL 3.0 in internet options, restarted and still doesn't seems to work, so i remembered that i used a program to disable them, used the revert, still not working

i tried clearing the ssl states, still nothing
maybe there's a way to add the osu ssl certificate directly? or i'm probably missing a windows component that i should enable

**Reply by peppy** [reply]
No idea sorry. We can't support such weird releases of windows, so you're on your own.
